command/root/root.go

```diff
@@ -0,0 +1,59 @@
+package root
+
+import (
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/command/backup"
+	"github.com/0xPolygon/polygon-edge/command/genesis"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/command/ibft"
+	"github.com/0xPolygon/polygon-edge/command/loadbot"
+	"github.com/0xPolygon/polygon-edge/command/monitor"
+	"github.com/0xPolygon/polygon-edge/command/peers"
+	"github.com/0xPolygon/polygon-edge/command/secrets"
+	"github.com/0xPolygon/polygon-edge/command/server"
+	"github.com/0xPolygon/polygon-edge/command/status"
+	"github.com/0xPolygon/polygon-edge/command/txpool"
+	"github.com/0xPolygon/polygon-edge/command/version"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+type RootCommand struct {
+	baseCmd *cobra.Command
+}
+
+func NewRootCommand() *RootCommand {
+	rootCommand := &RootCommand{
+		baseCmd: &cobra.Command{
+			Short: "Polygon Edge is a framework for building Ethereum-compatible Blockchain networks",
+		},
+	}
+
+	helper.RegisterJSONOutputFlag(rootCommand.baseCmd)
+
+	rootCommand.registerSubCommands()
+
+	return rootCommand
+}
+
+func (rc *RootCommand) registerSubCommands() {
+	rc.baseCmd.AddCommand(version.GetCommand())
+	rc.baseCmd.AddCommand(txpool.GetCommand())
+	rc.baseCmd.AddCommand(status.GetCommand())
+	rc.baseCmd.AddCommand(secrets.GetCommand())
+	rc.baseCmd.AddCommand(peers.GetCommand())
+	rc.baseCmd.AddCommand(monitor.GetCommand())
+	rc.baseCmd.AddCommand(loadbot.GetCommand())
+	rc.baseCmd.AddCommand(ibft.GetCommand())
+	rc.baseCmd.AddCommand(backup.GetCommand())
+	rc.baseCmd.AddCommand(genesis.GetCommand())
+	rc.baseCmd.AddCommand(server.GetCommand())
+}
```

Comment:
I think this can written in a single call to `AddCommand()`. It takes a varying number of commands.

Reply:
You're right, I've completely missed that the method signature allows for variable args 🤦 

Fixed for all commands 👍 

[865136e](https://github.com/0xPolygon/polygon-edge/pull/410/commits/865136e10fe121462c7177846b91353482fa652d)